edb/pgsql/metaschema.py

```diff
@@ -2591,6 +2672,7 @@ async def generate_support_functions(conn, schema):
         dbops.CreateFunction(IsinstanceFunction()),
         dbops.CreateFunction(GetSchemaObjectNameFunction()),
         # dbops.CreateFunction(QuoteLiteralFunction()),
```

Comment:
Remove this?

Reply:
Yes. That was in describe config PR, haven't rebased yet at that point.